assets/css/theme.css

```diff
@@ -128,44 +130,73 @@ h2 {
             explicitly set the aspect ratio of the svg, so that he height can be
             determined from the width of the ::after box height.
         */
```

Comment:
This file seems to have changed from 4 tab spaces -> 2

(This is correct for the workspace. There was an issue with older versions where some files were created as 4 tab spaces instead of 2)

Nit: These comments don't seem to have adjusting to the new formatting

Reply:
annoying the comments weren't updated as well